src/widget/sidebar.c

```diff
@@ -40,15 +41,17 @@
 #include "window/mission_briefing.h"
 #include "window/overlay_menu.h"
 
+#define SIDEBAR_SLIDE_STEPS 94
 #define SIDEBAR_COLLAPSED_WIDTH 42
 #define SIDEBAR_EXPANDED_WIDTH 162
-#define FILLER_Y_OFFSET 474
 
-#define EXTRA_INFO_HEIGHT_GAME_SPEED 64
-#define EXTRA_INFO_HEIGHT_UNEMPLOYMENT 112
-#define EXTRA_INFO_HEIGHT_RATINGS 272
+typedef struct {
+    time_millis slide_start;
+    int progress;
+    int focus_button_for_tooltip;
+} sidebar_data;
 
-#define SIDEBAR_SLIDE_STEPS 94
+sidebar_data sidebar_data_vals;
```

Comment:
ok, done - i had to leave 

```
typedef struct {                                                                                                                                                                                                   
    int value;                                                                                                                                                                                                     
    int target;                                                                                                                                                                                                    
} objective;                                                                                                                                                                                                       
 ```

above the function declarations though, because one of the functions uses objective in a parameter